php-checks/src/test/resources/checks/StringLiteralDuplicatedCheck/custom.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+echo "aa()";    // Noncompliant {{Define a constant instead of duplicating this literal "aa()" 5 times.}}
+echo "aa()";
+echo "aa()";
+echo "aa()";
+echo "aa()";
+
+echo "bbbb()";  // Noncompliant {{Define a constant instead of duplicating this literal "bbbb()" 4 times.}}
+echo "bbbb()";
+echo "bbbb()";
+echo "bbbb()";
```

Comment:
If we want to test the impact of `minimalLiteralLength = 4`, then I think we should have 2 sets of duplicated literals which differ only by the length of the literal. One compliant and one noncompliant.
```
echo "(((("; // Noncompliant {{Define a constant instead of duplicating this literal "((((" 5 times.}}
echo "((((";
echo "((((";
echo "((((";
echo "((((";

echo "((("; // Compliant
echo "(((";
echo "(((";
echo "(((";
echo "(((";
```
And I think we should do the same for `threshold`.